context/app/static/js/components/error/ErrorBody/ErrorBody.jsx

```diff
@@ -55,6 +55,17 @@ function ErrorBody({ errorCode, isAuthenticated, isGlobus401, isMaintenancePage
   }
 
   if (errorCode === 404) {
+    if (urlPath && urlPath.match(/^\/browse\//)) {
```

Comment:
Could use `startsWith`, but maybe regex is better?

Reply:
I'll go with `startsWith`: much easier to read, Thanks for the reminder!